src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -307,6 +302,32 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this.model.onScroll.fire();
 	}
 
+	public clickOffCell(e: MouseEvent) {
+		if (this.model.activeCell?.isEditMode) {
+			// when clicking outside of cells toggle edit mode to false
+			this.toggleEditMode();
+		} else {
+			this.unselectActiveCell();
```

Comment:
The current behavior in stable will toggle edit mode and unselect the active cell when you click outside the cell. Do we want to change the behavior to only toggle edit mode?

Reply:
Just reverted to the stable behavior